mount-helper-container/client/client.go

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ * Copyright 2024- IBM Inc. All rights reserved
+ * SPDX-License-Identifier: Apache2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"strings"
+)
+
+func main() {
+	// socket path
+	socketPath := "/tmp/mysocket.sock"
+	// payload
+	payload := fmt.Sprintf(`{"stagingTargetPath":"fs1235","targetPath":"/test","fsType":"ext4","requestID":"1321"}`)
```

Comment:
Why are these hardcoded. I believe these should be part of configurable items?. If this is a work in progress we could create a feature branch and go from there?

Reply:
This is fine as of now and it will be overwritten by using deployment file which will be as per configuration. This is initial code